content/0.index.md

```diff
@@ -9,13 +9,17 @@ main:
 :ellipsis{right=0px width=75% blur=150px}
 
 ::block-hero
+
 ---
+
 cta:
-  - Get started
-  - /introduction/getting-started
-secondary:
-  - Open on GitHub →
-  - https://github.com/dswan36/sillylittle.selfhost
+
+- Get started
+- /introduction/getting-started
+  secondary:
+- Open on GitHub →
+- https://github.com/dswan36/sillylittle.selfhost
+
```

Comment:
Link Breakage here.
Diff shows that it is possibly due to spaces introduced.
To Fix and analyze